contents/blog/company-survey-h1-2024.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2024-06-25
+title: What employees really think about PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/mykonoshackathon_979cf7bec6.jpeg
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Culture
+---
+
+At PostHog, we want to build the [most transparent company in the world](/handbook/people/overview) – and the best place for people to work in general. To find out how we’re doing, we run bi-annual team surveys. And to be totally transparent, we’re going to publish them here for anyone to see.
+
+We had a 79% response rate, or 33 out of 42 eligible respondents. For our last survey, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.
```

Comment:
```suggestion
We had a 79% response rate, or 33 out of 42 eligible respondents, for our H1 2024 survey. For our last survey, which covered Q4 2023, we had a 74% response rate. Since then, we’ve added 14 new team members and lost three. Ideally we’d get a response rate of 90% or higher, though we did have a good cross section of teams and tenures.
```